netket/operator/_sumoperators.py

```diff
@@ -29,7 +29,7 @@ class SumOperator(ContinuousOperator):
 
     def __init__(
         self,
-        *operators: List,
+        operators: List,
```

Comment:
This is breaking existing code (And you'd be breaking a lot of my internal code) so I would be in principle against this.
Why did you want to do this?

Reply:
ah but the operator test are all passing?

Reply:
ah yes I just replaced with [op1,op2,op3] but no need for that indeed